src/Tribe/Integrations/Manager.php

```diff
@@ -157,6 +159,16 @@ private function load_wp_rocket_integration() {
 		return true;
 	}
 
+	private function load_rcp_integration() {
+		if ( ! function_exists( 'rcp_user_can_access' ) ) {
+			return false;
+		}
+
+		tribe( RCP_Integration::class )->hook();
```

Comment:
Where is this Singleton registered? Shouldn't it be a provider?

Reply:
Yeah, that's an artifact of my changing the direction I went. I can fix that though. ❤️

Reply:
Here: https://github.com/the-events-calendar/the-events-calendar/pull/3948/files#diff-d954d62533e45649ed5be315a5d94782039510bc401eae4567ae2ee03aaaacc0R604